docs/src/main/asciidoc/kafka.adoc

```diff
@@ -249,6 +249,9 @@ The `channel-name` segment must match the value set in the `@Incoming` and `@Out
 
 [source]
 ----
+# Configure the SmallRye Kafka connector
+kafka.bootstrap.servers=localhost:9092
```

Comment:
Although this is the default and the example works without setting this, it is useful to point out so someone can get up and running connected to a different host or port quickly